tests/unit-tests/input/evdev/test_libinput_device.cpp

```diff
@@ -102,6 +94,18 @@ struct MockEventBuilder : mi::EventBuilder
                                        return builder.pointer_event(time, action, buttons, x, y,
                                                                     hscroll, vscroll, relative_x, relative_y);
                                   }));
+
+        ON_CALL(*this, pointer_axis_event(_, _, _, _, _, _, _, _, _, _))
+            .WillByDefault(
+                Invoke(
```

Comment:
To reduce clutter, you could drop the `Invoke()` from here. Rest is fine AFAICT.